packages/integration-tests/index.js

```diff
@@ -449,54 +553,39 @@ async function addProviders(stackName, bucketName, dataDirectory, s3Host, postfi
   return completeProviders.length;
 }
 
-/**
- * Return a list of providers
- *
- * @param {string} stackName - Cloud formation stack name
- * @param {string} bucketName - S3 internal bucket name
- * @param {string} dataDirectory - the directory of provider json files
- * @returns {Promise.<Array>} list of providers
- */
-async function listProviders(stackName, bucketName, dataDirectory) {
-  return setupSeedData(stackName, bucketName, dataDirectory);
-}
-
 /**
  * Delete providers from database
  *
  * @param {string} stackName - CloudFormation stack name
- * @param {string} bucketName - S3 internal bucket name
  * @param {Array} providers - List of providers to delete
  * @param {string} postfix - string that was appended to provider id
- * @returns {Promise.<number>} number of deleted providers
+ * @returns {Promise<number>} number of deleted providers
  */
-async function deleteProviders(stackName, bucketName, providers, postfix) {
-  setProcessEnvironment(stackName, bucketName);
-
-  const promises = providers.map((provider) => {
-    if (postfix) {
-      provider.id += postfix;
-    }
-    const p = new Provider();
-    console.log(`Deleting provider ${provider.id}`);
-    return p.delete(provider);
-  });
+async function deleteProviders(stackName, providers, postfix) {
+  await Promise.all(
+    providers.map(
+      ({ id }) => {
+        const readId = postfix ? `${id}${postfix}` : id;
+        return providersApi.deleteProvider(stackName, readId);
+      }
+    )
+  );
 
-  return Promise.all(promises).then((ps) => ps.length);
+  return providers.length;
 }
 
 /**
  * Delete all collections listed from a collections directory
  *
  * @param {string} stackName - CloudFormation stack name
- * @param {string} bucket - S3 internal bucket name
+ * @param {string} _bucket - S3 internal bucket name
```

Comment:
Why the underscore?

Reply:
The bucket variable is never used. The `no-unused-vars` eslint rule complains about that but, in our eslint config, (we allow overriding that for variables that begin with an underscore)[https://github.com/nasa/cumulus/blob/master/.eslintrc.json#L71].

In this case the better solution would be to remove that variable entirely, but this function is called all over the place and, to keep the PR from exploding, I chose to just not update all of them.